post-processor/shell-local/post-processor.go

```diff
@@ -68,15 +69,18 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 	}
 
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = `chmod +x "{{.Script}}"; {{.Vars}} "{{.Script}}"`
+		p.config.ExecuteCommand = `{{.Script}}`
 	}
 
 	if p.config.Inline != nil && len(p.config.Inline) == 0 {
 		p.config.Inline = nil
 	}
 
-	if p.config.InlineShebang == "" {
-		p.config.InlineShebang = "/bin/sh -e"
+	if p.config.InlineShebang == nil {
+		p.config.InlineShebang = []string{"/bin/sh", "-e"}
+		if runtime.GOOS == "windows" {
+			p.config.InlineShebang = []string{"sh", "-e"}
```

Comment:
I don't understand what's going on here. is `sh` a valid windows command? Does windows command/powershell understand what a bang line is?

Reply:
`/bin/sh` isn't valid because that isn't a valid windows path and isn't where windows stores `sh` . I can try to find the absolute system path, but at least on windows 10 this works for the inline scripts.

Reply:
it's _probably_ not needed